opencensus/log.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2019, OpenCensus Authors
```

Comment:
We need to put this inside a child directory.
`./opencensus/` is a namspace path, log.py will get removed if user uninstalls other packages like opencensus-ext-mysql.

Reply:
Right, I started this branch before your changes landed and hadn't updated the paths yet. Fixed in 24fea93.